gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/jupiter/core/context/AbstractResponse.java

```diff
@@ -79,7 +80,7 @@ public boolean ended() {
     }
 
     @Override
-    public Completable end() {
+    public Completable end(final GenericExecutionContext ctx) {
```

Comment:
Because VertxHttpServerResponse implementation needs it to compute the response content length while chunks are consumed. (see line 59 of VertxHttpServerResponse)